docs/AppTutorial.md

```diff
@@ -0,0 +1,461 @@
+
+![alt tag](./images/StaticComponent.png)
+
+As we have already added the *Heading* component in previous steps, we will now be adding *Text* component. In *CovidInfo/index.js* :
+```jsx
+import { Text } from 'design-system';
+```
+We can use *Text* component like this:
+```jsx
+<div>
+    <Text>Use a medical face mask</Text>
+    <Text>Wash your hands with soap and water</Text>
+</div>
+```
+Similarly, Symptoms can be added in the following way:
+```jsx
+<div>
+    <Heading>Symptoms</Heading>
+    <div>
+        <Text>Fever, Cough</Text>
+        <Text>Respiratoy Symptoms</Text>
+    </div>
+</div>
+```
+You can style the CovidInfo component by adding CSS in *CovidInfo.css*
+
+### Building Dynamic Components
+This step takes our components and populates them with data from the  GraphQL API – loading states and all. To do so, we’ll be using Apollo Client, the front-end component of the [Apollo Platform](https://www.apollographql.com/docs/intro/platform). Apollo Client is a sophisticated GraphQL client that manages data and state in an application.
+You need to follow these steps to fetch data from API:
+* ###### Install dependencies
+    
+    We’ll need to install three new dependencies to incorporate Apollo into our application namely `apollo-boost`, `graphQl` and `react-apollo`
+* ###### Connect to Apollo
+    Create a new directory src/api.
+    ```jsx
+    src
+    ├──api
+    ├──index.js
+    ```
+    Inside *api/index.js*, you can make your client by providing a URI :
+    ```jsx
+    import ApolloClient from 'apollo-boost';
+    export const client = new ApolloClient({
+        uri: 'https://covid-tracker-news-graphql.now.sh/'
+    });
+    ```
+    Now we need to wrap our application with ApolloProvider. So, inside *src/index.js* :
+    ```jsx
+    import { ApolloProvider } from '@apollo/react-hooks';
+    import { client } from './api';
+    ReactDOM.render(
+        <ApolloProvider client={client}>
+            <Router>...</Router>
+        </ApolloProvider>
+    )
+    ```
+* ###### Fetch data
+    Next we’ll assemble our GraphQL using the gql helper. The gql helper lets you write GraphQL queries using interpolated strings (backticks) in JavaScript. In addition, we’ll be using the Query component from react-apollo which gives us some great information about our query’s loading state in addition to the data. [Click here](https://github.com/sandeshchoudhary/covid/blob/master/src/query.js) for more details. 
+
+Finally, you can get the data, loading and error states in your component using `useQuery` from `@apollo/react-hooks`. For example: 
+```jsx
+const { loading, error, data } = useQuery(query.data);
+```
+Now that we have fetched the required data, we can easily render them in our component. Now its time to build `Map component` using the fetched API data. Here is the preview of what you will build:
```

Comment:
Highlight or add a heading to inform we are going to use visualization components.